### PR TITLE
ns-process-data: fix image renaming for the --skip-colmap case

### PR DESCRIPTION
**Pull request by jkulhanek**
This PR (hopefuly) fixes problems introduced in #1366 and partially fixed in ( #1454 ). I am sorry for the problems introduced by the first PR.

@tancik , can you please take a look? This should fix the problems in the original PR while adding back the functionality of (--skip-colmap).

### CODE REVIEW

File: nerfstudio/process_data/process_data_utils.py [hunk 1]
```diff
@@ -241,7 +241,7 @@ def copy_and_upscale_polycam_depth_maps_list(
     return copied_depth_map_paths
 
 
-def copy_images(data: Path, image_dir: Path, verbose) -> int:
+def copy_images(data: Path, image_dir: Path, verbose) -> OrderedDict[Path, Path]:
     """Copy images from a directory to a new directory.
 
     Args:
```
Comment (tancik) [inline nerfstudio/process_data/process_data_utils.py:247]: Update `Returns` doc

**Review by tancik — COMMENTED** [review]
Tested, seems to be working. Thanks for the quick fix.

File: scripts/process_data.py [hunk 2]
```diff
@@ -129,7 +131,11 @@ def main(self) -> None:
         # Copy and downscale images
         if not self.skip_image_processing:
             # Copy images to output directory
-            num_frames = process_data_utils.copy_images(self.data, image_dir=image_dir, verbose=self.verbose)
+            image_rename_map_paths = process_data_utils.copy_images(
+                self.data, image_dir=image_dir, verbose=self.verbose
+            )
+            image_rename_map = OrderedDict((a.name, b.name) for a, b in image_rename_map_paths.items())
```
Comment (tancik) [inline scripts/process_data.py:137]: Isn't `image_rename_map_paths` already an `OrderedDict`?

Comment (jkulhanek) [reply scripts/process_data.py:137]: It is. Here the rename_map takes the filenames as strings.

Comment (brentyi) [reply scripts/process_data.py:137]: just wondering: why not use a regular dict?

Comment (jkulhanek) [reply scripts/process_data.py]: No reason, switching to dict.

**Review by tancik — APPROVED** [review]
Lgtm! Thanks for fixing.

**Pull request merged**